### DISCUSSION TITLE
Selection Filtering

### DISCUSSION THREAD

**Discussion by RobertJoonas** [plausible/analytics#1600] [Anything Else]
In the dashboard, I want to look at my stats with a country filter on multiple countries. Would be nice if the "add filter" functionality would include a logical OR filter or a selection, so that for example, I could look at all visits from France and Germany at the same time.

The same filtering functionality could be used for pages, referral sources, browsers, etc too.

**Comment by ukutaht** [comment]
Agreed. Related discussion: https://github.com/plausible/analytics/discussions/1528
